controls/radchartview/axes/datetimeaxes.md

```diff
@@ -10,24 +10,36 @@ position: 2
 
 # Date Time Axes
 
+__RadChartView__ has two DateTime dedicated axes. The first one is __DateTimeCategoricalAxis__, the other is __DateTimeContinuousAxis__. These axes are different in nature as one is categorical (discrete), and the other is continuous. This topic will cover the specific properties of these two axes.
 
+## __Date Time Continuous Axis__
 
-## 
+The __DateTimeContinuousAxis__ is like a numerical axis but the axis value range consists of DateTime values which are always sorted chronologically. This axis expects each data point to provide a System.DateTime structure as its value for this axis. It does not work with categories, it works with ranges. The base unit (or the timeline step) of the axis is calculated depending on the smallest difference between any two dates.
 
-__Date Time Continuous Axis__
+### Specific Properties
+
+* __ActualRange__: A property of type __ValueRange<DateTime>__ that gets the actual range used by the axis to plot data points.

```

Comment:
In general I would recommend avoiding lists of properties. This is because this information can be very easily found in the API Reference or by pressing F2 over the corresponding class in Visual Studio. Usually, the descriptions we use for the properties are the same as the ones in the class' code. 

Instead I could recommend using sections or bullets showing the different properties with a single line of code that sets them and a picture showing their result. Such example could be found in the [ChartView3D Axes](https://docs.telerik.com/devtools/wpf/controls/radchartview3d/axes) article.

For example the list with the specific properties could be organized using the following structure:
- Setting the Axis Range Manually - this could be a section, a list bullet or some other unit. And it could describe the Minimum, Maximum, MajorStep and MajorStepUnit, MaximumTicks and TickOrigin properties.
- Setting the PlotMode - this contain a single sentence on what PlotMode is + link to the article
- Setting the GapLength - single sentence + link to the article
- Getting the Axis Range - here you can tell about the ActualVisibleRange

Anyway, this is merely a feedback and it is only optional. Currently, you can leave this part as it is.